build-system/server/new-server/router.js

```diff
@@ -2,8 +2,14 @@ const router = require('express').Router();
 // @ts-ignore
 const {transform} = require('./transforms/dist/transform');
 
-router.get('/examples/*.html', async (req, res) => {
+router.get('/examples/*.html', async (req, res, next) => {
   let transformedHTML;
+  // nomodule.html will force an opt-out of the transformation. This can
+  // be useful if we want to explicitly test the nomodule version of the
+  // code.
+  if (/\.nomodule\.html$/.test(req.path)) {
```

Comment:
This feels a bit hackish and difficult to discover, I wonder if there's a way to be more explicit about this. e.g., can we add an HTTP header (something like `X-AMP-No-Module`) and explicitly add this to the .jsonc file as a field? (`force-nomodule: true`)

Otherwise if we decide to stick with this method in the end:
```suggestion
  if (req.path.endsWith('.nomodule.html')) {
```

Reply:
i like the headers idea. ill implement